docs/howtos/signing-android-applications.md

```diff
@@ -1,36 +0,0 @@
-> Android requires that all APKs be digitally signed with a certificate before they can be installed.
```

Comment:
Maybe controversial, but...I don't think this doc belongs here. It's specifically about generating apks, which we don't do in this repo, so it wasn't obvious how to link to it from the ToC.

I wonder if there's some more android-product-development-centric place we could move it to?

Reply:
Perhaps this could live in https://github.com/mozilla-mobile/shared-docs?  @vladikoff thoughts?

Reply:
Ref https://github.com/mozilla-mobile/shared-docs/issues/80 for following

Reply:
Ah this doc was useful before, we can move it though, it's okay 👍

Reply:
Yep, I don't think we should lose it entirely, it's 👍 content